profiles/fscloud/variables.tf

```diff
@@ -0,0 +1,146 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to use"
+  type        = string
+  sensitive   = true
+}
+
+# Resource Group Variables
+variable "resource_group_id" {
+  type        = string
+  description = "The Id of an existing IBM Cloud resource group where the cluster will be grouped."
+}
+
+variable "region" {
+  type        = string
+  description = "The IBM Cloud region where the cluster will be provisioned."
+}
+
+# Cluster Variables
+variable "tags" {
+  type        = list(string)
+  description = "Metadata labels describing this cluster deployment, i.e. test"
+  default     = []
+}
+
+variable "cluster_name" {
+  type        = string
+  description = "The name that will be assigned to the provisioned cluster"
+}
+
+variable "vpc_subnets" {
+  type = map(list(object({
+    id         = string
+    zone       = string
+    cidr_block = string
+  })))
+  description = "Metadata that describes the VPC's subnets. Obtain this information from the VPC where this cluster will be created"
+}
+
+variable "worker_pools" {
+  type = list(object({
+    subnet_prefix     = string
+    pool_name         = string
+    machine_type      = string
+    workers_per_zone  = number
+    resource_group_id = optional(string)
+    labels            = optional(map(string))
+    boot_volume_encryption_kms_config = optional(object({
+      crk             = string
+      kms_instance_id = string
+      kms_account_id  = optional(string)
+    }))
+  }))
+  description = "List of worker pools"
+}
+
+variable "worker_pools_taints" {
+  type        = map(list(object({ key = string, value = string, effect = string })))
+  description = "Optional, Map of lists containing node taints by node-pool name"
+  default     = null
+}
+
+variable "ignore_worker_pool_size_changes" {
+  type        = bool
+  description = "Enable if using worker autoscaling. Stops Terraform managing worker count"
+  default     = false
+}
+
+variable "ocp_version" {
+  type        = string
+  description = "The version of the OpenShift cluster that should be provisioned (format 4.x). This is only used during initial cluster provisioning, but ignored for future updates. If no value is passed, or the string 'latest' is passed, the current latest OCP version will be used."
+  default     = null
+
+  validation {
```

Comment:
does the validation from the main module carry over here? Just thinking maybe remove it to avoid having to maintain in 2 places?